code/game/jobs/jobs.dm

```diff
@@ -118,6 +118,13 @@ var/list/security_positions = list(
 )
 
 
+var/list/planet_positions = list(

```

Comment:
Apparently there's gonna be planets eventually so remind me when that time comes to remap the planets and also probably change their names because Polaris will probably hard code it.